kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -132,14 +131,12 @@
       const { genExternalContentURLBackLinkCurrentPage } = useContentLink();
       const { getLearningActivityIcon } = useLearningActivities();
       const { downloadRequestMap, availableSpace } = useDownloadRequests();
-      const totalPageNumber = ref(0);
       const store = getCurrentInstance().proxy.$store;
       const query = computed(() => get(route).query);
       const route = computed(() => store.state.route);
       const sort = computed(() => query.value.sort);
       const pageSizeNumber = computed(() => Number(query.value.page_size || 25));
       const activityType = computed(() => query.value.activity || 'all');
-      const downloads = ref([]);
       const sortedFilteredDownloads = () => {
```

Comment:
I think my suggestion was to turn this into a computed prop, but this is still just a plain function.